pkg/logger/logger.go

```diff
@@ -145,6 +148,25 @@ func (l *Logger) Output(calldepth int, message string) {
 	if err != nil {
 		panic(err)
 	}
+	logBuff.Write([]byte("\n"))
+
+	return logBuff.Bytes()
+}
+
+// Output a standard log template with a simple message to default output channel.
+// Write a final newline at the end of every message.
+func (l *Logger) Output(lvl Level, calldepth int, message string) {
+	l.mu.Lock()
+	defer l.mu.Unlock()
+	if !l.stdEnabled {
+		return
+	}
+	msg := l.formatLogMessage(calldepth, message)
+	if lvl == DEFAULT {
+		l.writer.Write(msg)
+	} else { // currently we only have info and error levels
+		l.errWriter.Write(msg)
+	}
```

Comment:
I think it would be preferable to be more explicit that DEFAULT is the default, so I would make this a switch like so
```suggestion
	switch lvl {
	case ERROR:
		l.errWriter.Write(msg)
	default:
		l.writer.Write(msg)
	}
```